rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -25,6 +25,22 @@ spec:
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
             runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/sre/core-pipeline-controller-repeated-restarts.md
+        - alert: PipelinePodsCrashLoopBackOff
+          expr: |
+            sum by (source_cluster) (max_over_time(kube_pod_container_status_waiting_reason{namespace="openshift-pipelines", reason="CrashLoopBackOff"}[3m]) or vector(0)) > 0
```

Comment:
Could we please add one more filter here to only alert when the main pods are not in a good status? The namespace has other pods and based on the comments [here](https://issues.redhat.com/browse/KONFLUX-3581?focusedId=25192463&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-25192463) we are interested in the controllers started with tekton or pipeline as code. Please let me know if this does not apply anymore.